Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -165,74 +165,28 @@ Use ``--verbose`` for full details:
       - HostPort:       Enabled
     [...]
 
-As a optional next step, we deploy nginx pods, create a new NodePort service and
-validate that Cilium installed the service correctly.
+As a optional next step, we deploy the connectivity checks and validate that
```

Comment:
One thing which I like about the previous validation check that it's very simple, no hidden details, and users can learn how Cilium represents their services internally. OTOH, because of its simplicity, it lacks more extensive validation. Anyway, I don't have strong opinion here. Maybe @borkmann does have.

One thing which we should eventually address in the connectivity check suite, is that it doesn't check service implementation in bpf_host.c, as all SVC requests are handled by bpf_sock.c

Reply:
> One thing which I like about the previous validation check that it's very simple, no hidden details, and users can learn how Cilium represents their services internally.

I understand that. It's also why I kept the `cilium service list` step: it's short and familiarize users with that command and Cilium's internal representation. Nonetheless, in general I expect getting started guides to provide a _short_ way for the user to get the setup they're looking for. If we want to _demo_ that setup, they maybe that's a better fit for a blog post (e.g., our K3s post) or a tutorial (e.g., our L3-L7 policy tutorial).

Reply:
Hm, I'm similar opinion as @brb on this one. Too much magic happening in the background in the `k8s-install-connectivity-test.rst` . I personally haven't used the latter for manual testing either. The below example builds upon earlier my-nginx example yaml from the quick intro and for testing I'm usually deploying this snippet locally. Maybe we could refer to the `k8s-install-connectivity-test.rst` at the end for a more elaborate connectivity check?

Reply:
I'd rather not make this guide even longer. I've removed the last two commits from this PR, to only keep the minor syntax/case changes.